addons/crm/static/src/views/forecast_kanban/forecast_kanban_renderer.xml

```diff
@@ -3,13 +3,26 @@
     <t t-name="crm.ForecastKanbanRenderer" t-inherit="crm.CrmKanbanRenderer" t-inherit-mode="primary" owl="1">
         <KanbanColumnQuickCreate position="replace">
             <t t-if="isGroupedByForecastField()">
-                <ForecastKanbanColumnQuickCreate
-                    folded="true"
-                    onFoldChange="() => {}"
-                    onValidate.bind="addForecastColumn"
-                    exampleData="exampleData"
-                    groupByFieldString="getForecastQuickCreateTitle()"
-                />
+                <t t-if="canApplyExampleData()">
+                    <ForecastKanbanColumnQuickCreate
+                        folded="true"
+                        onFoldChange="() => {}"
+                        onValidate.bind="addForecastColumn"
+                        exampleData="exampleData"
+                        canApplyExampleData="true"
```

Comment:
Since we're folded anyway, this can just be false every time (avoids having to call the function in here)

Although, you say you got undefined so it was opened?

Reply:
ok thanks, i will set it to false, as it's folded